utils/formatDateRange.ts

```diff
@@ -0,0 +1,12 @@
+import { format as dateFnsFormat, parseISO as parseDate, addHours } from 'date-fns';
+
+const formatDateRange = (startsAt: string, workhour: number) => {
```

Comment:
P5: 여기도 `workHour`로 수정하면 좋을 것 같아요!!

```suggestion
const formatDateRange = (startsAt: string, workHour: number) => {
```

Reply:
이건 수정해 놓겠습니다